public/app/features/integrated-alerting/components/AlertRuleTemplate/AlertRuleTemplatesTable/AlertRuleTemplatesTable.styles.ts

```diff
@@ -0,0 +1,85 @@
+import { css } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+import { selectThemeVariant, stylesFactory } from '@grafana/ui';
+
+export const getStyles = stylesFactory((theme: GrafanaTheme) => {
+  const backgroundColor = selectThemeVariant({ light: 'rgb(247, 247, 249)', dark: '#161719' }, theme.type);
+  const borderColor = selectThemeVariant({ light: theme.palette.gray85, dark: '#292929' }, theme.type);
+  const headerBackground = selectThemeVariant({ light: 'rgb(247, 247, 249)', dark: '#3D3D3D' }, theme.type);
+
+  return {
+    /* This will make the table scrollable when it gets too small */
+    tableWrap: css`
+      border: 1px solid ${borderColor};
+      display: block;
+      max-width: 100%;
+      max-height: 500px;
+      overflow-x: auto;
```

Comment:
nit: shortcut for both is `overflow: auto;`

Reply:
done